catalog-spec/catalog-recommendations.md

```diff
@@ -61,7 +142,7 @@ of canonical product definitions online, that all static catalogs using a partic
 able to reference.
 
 
-## Product
+### Product
```

Comment:
Products are outdated => Dataset

Reply:
Well I think product was more just supposed to capture the asset level metadata, not the dataset level stuff. I agree that it should be added at the dataset 'level', but we don't have the clear proposal for how to do that. I'm not sure if we should require binding it to the dataset level - at Planet we have a lot of assets and it may make sense for an Item's asset to refer to its 'product definition' (asset metadata) dynamically, instead of saying it needs to be enumerated at the dataset level.

Though not sure if this is the best place to discuss this - this PR is just an attempt to make various bits better...